CHANGELOG.md

```diff
@@ -4,7 +4,21 @@
 
 This is a feature release.
 
+ * [KIP-951](https://cwiki.apache.org/confluence/display/KAFKA/KIP-951%3A+Leader+discovery+optimisations+for+the+client):
+   Leader discovery optimisations for the client.
+ * [KIP-714](https://cwiki.apache.org/confluence/display/KAFKA/KIP-714%3A+Client+metrics+and+observability):
+   Client metrics and observability.
```

Comment:
These are already present in the librdkafka changelogs. I think we should remove these from here.

Reply:
Done, let's discuss this for the next release later